app/scripts/views/confirm_account_unlock.js

```diff
@@ -137,9 +137,12 @@ function (Cocktail, FormView, BaseView, Template, p, AuthErrors, Constants,
           email,
           password,
           self.relier,
+          // TODO: remove once https://github.com/eslint/eslint/issues/1801 gets fixed
```

Comment:
nit: comment not consistent with other comments, should look more like the ones we have before `See eslint/eslint#1801`